incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -165,17 +166,36 @@ func TestNonRBACTypes(t *testing.T) {
 
 func TestPropagateConflict(t *testing.T) {
 	tests := []struct {
-		name         string
-		inNamespaces string
-		allow        bool
+		name   string
+		forest string
+		// inNamespace contains the namespaces we are creating the objects in,
+		inNamespace   string
+		noPropogation string
```

Comment:
Looks like that comment needs to be finished? A comment for `noPropagation` might not be a bad idea either but that's a nit, it's _fairly_ self-explanatory.

Reply:
done